integr8ly/integreatly-operator
# Silence Alerts for Keycloak on uninstall

# Description
Silence critical keycloak alert on uninstall. 
https://issues.redhat.com/browse/MGDAPI-1612

## Type of change




## Verification

1. Install RHOAM or RHMI from this branch
2. Scale down keycloak operator and keycloak pods
3. verify alert is firing in openshift-monitoring alertmanager and AMO alertmanager
4. Start uninstall 
5. verify silence gets created in both alertmanagers
